formats/placeholder/frontend/schema.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "base_path",
+    "format",
+    "locale"
+  ],
+  "properties": {
+    "base_path": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "format": {
+      "type": "string",
+      "enum": [
+        "placeholder"
```

Comment:
This will mean that any placeholder items with the prefix format (e.g. "placeholder_organisation") will not pass validation. Can this be made a pattern match instead?